worker/caasoperator/action.go

```diff
@@ -87,27 +85,27 @@ func prepare(
 	serviceAddress string,
 	operatorPaths Paths,
 	unitPaths uniter.Paths,
+	operatorInfo params.OperatorInfo,
 	stdout io.ReadWriter,
 	stderr io.Writer,
 	cancel <-chan struct{},
 ) error {
 	// TODO(caas) - quick check to see if files have already been copied across.
 	// upgrade-charm and upgrade-juju will need to ensure files are up-to-date.
 	operatorFile := filepath.Join(unitPaths.State.BaseDir, provider.OperatorInfoFile)
+	var o bytes.Buffer
```

Comment:
s/o/stdout
or outbuf

Reply:
Haha I'd like to point out who named this variable https://github.com/juju/juju/commit/d780faaf774bddd0b904e4428ae6c33076683f55#diff-45dba74aed2232e6a95ec5e8c580fab8R108